components/x-timeline-feed/src/lib/date.js

```diff
@@ -0,0 +1,51 @@
+import { format, isAfter, differenceInCalendarDays, subMinutes } from 'date-fns';
```

Comment:
In x-teaser we're using the `dateformat` module, would be good to see if we can standardise on one or the other.

Reply:
Makes sense. I'd favour standardising on [date-fns](https://date-fns.org/) as it provides many date functions, including formatting.